package.json

```diff
@@ -46,6 +46,7 @@
     "lib"
   ],
   "dependencies": {
+    "@hapi/hoek": "^9.1.1",
```

Comment:
Outside the scope of this PR, but didn't the hapi modules get descoped in transitioning the project over to new maintainers?  Any updates on this?

Reply:
It's a bit of a confusing situation. Only `joi` transitioned back out of the `hapi` umbrella. The rest of the modules, including `hoek`, stayed under the Hapi umbrella. You can still find it on the official site [here](https://hapi.dev/module/hoek), and I'll link to the official announcement if I can find it.

Reply:
Ah, I see.  

I'll work on migrating `@hapi/joi` back `joi`.